ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -529,6 +532,21 @@ private static ComorbidityDetails ExtractComorbidityDetails(MigrationDbNotificat
             return details;
         }
 
+        private async Task<ICollection<PreviousTbService>> ExtractPreviousTbServices(IEnumerable<TreatmentEvent> notificationTransferEvents)
+        {
+            var tbServices = await _referenceDataRepository.GetAllTbServicesAsync();
```

Comment:
I have move the initialisation of the tbService here to match the refactoring done in NTBS 2966